components/forms/DynamicRow/DynamicRow.tsx

```diff
@@ -50,67 +50,57 @@ export const DynamicGroup = (props: DynamicGroupProps): React.ReactElement => {
     maxNumberOfRows,
   } = props;
   const [field, meta, helpers] = useField(props);
-  const [rows, setRows] = useState([rowElements]);
-  const [rowRefs, setRowRefs] = useState<Array<React.RefObject<HTMLFieldSetElement>>>([]);
-  const [focussedRow, setFocussedRow] = useState<number | null>(null);
+  const [rows, setRows] = useState(() => Array(field.value.length).fill(rowElements));
+  const rowRefs = useRef<Array<React.RefObject<HTMLFieldSetElement>>>(
+    Array(field.value.length).fill(createRef<HTMLFieldSetElement>())
+  );
+  const focussedRow = useRef<number | null>(null);
```

Comment:
nit (feel free to ignore): I wonder if we can correct the spelling here for clarity? If we do, we'll have to update the other places in the code as well
```suggestion
  const focusedRow = useRef<number | null>(null);
```

Reply:
Not a nitpick.  You'll learn I'm horrible in missing spelling mistakes.  I'll fix tomorrow and update the PR.  Thanks for the catch.